modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -31,16 +31,16 @@
 	"all-projects": "All projects",
 	"allowed-email-domains": "Allowed Email Domains",
 	"almost-there-setup-analytics-cloud-by-finishing-the-activation-form": "Almost there! Setup Analytics Cloud by finishing the activation form.",
-	"almost-there-setup-experience-cloud-by-finishing-the-activation-form": "Almost there! Setup Experience Cloud by finishing the activation form.",
-	"almost-there-setup-lxc-sm": "Almost there! Setup LXC - SM",
+	"almost-there-setup-experience-cloud-by-finishing-the-activation-form": "Almost there! Setup Liferay SaaS by finishing the activation form.",
```

Comment:
I don't think Brian is going to allow the lang key being different from the lang value (eg `experience-cloud=Liferay Saas`). While variable names can stay the same, lang keys won't make it past Brian's review. Can you please update all the lang key values as well

Reply:
Okay, done.